src/linker/Linker/Driver.cs

```diff
@@ -1115,11 +1115,9 @@ static string[] ReadLines (string file)
 			return null;
 		}
 
-		AssemblyRootMode? ParseAssemblyRootsMode (string s)
+		AssemblyRootMode? ParseAssemblyRootMode (string s)
 		{
 			switch (s.ToLowerInvariant ()) {
-			case "default":
```

Comment:
Is there any chance we get this on the command line?
Just thinking about backward compat.

Reply:
I think it's OK to break this as well - especially since this is 8.0 only and we will have TFM-specific linker then.